docs/user/02-operation-guides/operations/02-10-check-if-sidecar-injection-is-enabled.md

```diff
@@ -0,0 +1,82 @@
+# Check if you have Istio sidecar proxy injection enabled
+
+## Check if sidecar injection is enabled in the Pod's Namespace
+
+<!-- tabs:start -->
+
+#### **kubectl**
```

Comment:
VV will hate that Dashboard isn't the first option :D 

Which method do we recommend? We should put that one first. If kubectl is better/recommended, keep it as first option

Reply:
We recommend kubectl because it is more straightforward for operations/troubleshooting (one command = searching + a few clicks in the UI). Let's keep it as the first option

Reply:
Agreed!